llmfoundry/tokenizers/tiktoken.py

```diff
@@ -61,6 +61,7 @@ def __init__(self,
                  eos_token: Optional[str] = '<|endoftext|>',
                  bos_token: Optional[str] = '<|endoftext|>',
                  pad_token: Optional[str] = None,
+                 errors: str = 'replace',
```

Comment:
nit: rename to `decode_errors`

Reply:
Any reason to make this non configurable for now and directly do `decode('utf-8', errors='replace')`? Generally thinking it might be helpful to opt for less configurability with apis--less room for user error and what not

Reply:
I'd like to leave for consistency because this is actually just copied from https://github.com/huggingface/transformers/blob/80377eb018c077dba434bc8e7912bcaed3a64d09/src/transformers/models/gpt2/tokenization_gpt2.py#L136. not a strong opinion though, wdyt?

Reply:
ahh got it, don't feel too strongly about it. I'm okay to leave it